docs/release_notes.md

```diff
@@ -1,5 +1,113 @@
 #
 
+## 4.3.2
+
+Release Date: 2024-04-xx
+
+This release represents preparation for work on 5.0.0 and important bug fixes for the 4.3 series.
+
+Most significantly, this release includes the new GenQuery2 parser and new iCommand `iquery`.  For anyone already running the unreleased GenQuery2 API plugin, please note that it will halt and report a conflict with 4.3.2.  Please uninstall the separate package before upgrading to 4.3.2.
```

Comment:
That's what I forgot to do - adjust the cmake in the GenQuery2 repo to disallow compiling against 4.3.2.

Will get that in before this is published.

What do mean by `it will halt and report a conflict with 4.3.2`?

Reply:
That was in the linked text... https://github.com/irods/irods/issues/7570#issuecomment-2043304940

Open to better wording, of course.

Reply:
how about...

"For anyone already running the unreleased GenQuery2 API plugin, please note that an upgrade will halt and report a conflict with 4.3.2."

Reply:
Wait. That statement is true. 4.3.2 declares a conflict with GenQuery2 packages.

What you have is good/correct.

Is it clearer to use `GenQuery2 packages` instead of `GenQuery2 API plugin`?

Reply:
mmmm, i shall combine them...

"For anyone already running the unreleased GenQuery2 API plugin packages, please note that an upgrade will halt and report a conflict with 4.3.2."

Reply:
added commit to tweak.

Reply:
Sounds good.